kie-osgi/kie-karaf-features/src/main/filtered-resources/repository/features-core.xml

```diff
@@ -35,6 +35,7 @@
 
   <feature name="kie" version="${project.version}" description="KIE API" start-level="70">
     <feature version="${project.version}">drools-common</feature>
+    <bundle>mvn:org.kie.soup/kie-soup-xstream/${version.org.kie}</bundle>
     <bundle>mvn:org.kie.soup/kie-soup-commons/${version.org.kie}</bundle>
```

Comment:
Please notice bundle kie-soup-commons is still required, as it's
required by bundles kie-soup-project-datamodel-api and
kie-soup-project-datamodel-commons